patches/minecraft/net/minecraft/world/level/Level.java.patch

```diff
@@ -206,9 +207,13 @@
                      blockstate = this.getBlockState(blockpos);
 -                    if (blockstate.is(Blocks.COMPARATOR)) {
 +                    if (blockstate.getWeakChanges(this, blockpos)) {
++                        // Forge: Replace Level#neighborChanged with BlockState#onNeighborChange. Boolean.valueOf(false) is used to account for problematic mixins (see #10402).
++                        if (Boolean.valueOf(false))
```

Comment:
We don't need this

Reply:
Ok, will remove later.

Reply:
I just tested this change on 1.20.1 on some big modpacks. The config value was off, but if there was a Mixin issue, the game would have crashed. So yeah, this is fine. The weird one in 1.20.1 for villager types was just an unfortunate case.